Add: Release announcement for 14.0

TrueBrain opened the pull request:
![Openttd_14_release](https://github.com/OpenTTD/website/assets/68320206/acf1e8ec-b9a5-4fd3-8e7b-6f2cd79e5a09)
[Openttd_14_release.zip](https://github.com/OpenTTD/website/files/14942565/Openttd_14_release.zip)

Steam:
```
Welcome to 14.0!

OpenTTD's first release was in March of 2004.
Now twenty years later, we are proud to present to you: release 14.0.

<image>

And boy, what a release it is.
Do I dare to say: this has been the biggest release yet?

Sometimes people try to tell us OpenTTD is dead.
Well, 14.0 shows you it really isn't.
There are so many new goodies in 14.0, that I don't even know where to start!
Let's go over a few, just to get you a feel for what to expect when you launch OpenTTD 14.0.
```

Discord / Reddit / Mastodon / ...
```
Welcome to 14.0!

OpenTTD's first release was in March of 2004.
Now twenty years later, we are proud to present to you: release 14.0.

And boy, what a release it is.
Do I dare to say: this has been the biggest release yet?

https://www.openttd.org/news/2024/04/13/openttd-14-0

<image>
```

Only do the last two lines if the social platform actually supports attaching images :)

James103 commented on _posts/2024-04-01-openttd-14-0.md:
```diff
@@ -0,0 +1,119 @@
+But once done, OpenTTD can talk to platforms like Steam, Discord and GOG Galaxy.
+
+For now, the functionality is limited: friends can see you play OpenTTD, if you are in a server or not, and what kind of map you are playing.
+For future versions we intend to expand on that, making more possible.
```
```suggestion
For future versions, we intend to expand on that, making more possible.
```

James103 commented on _posts/2024-04-01-openttd-14-0.md:
```diff
@@ -0,0 +1,119 @@
+Bringing information closer to the user, and the game more accessible to everyone, has been a path we have been on for a while.
+
+Second, we revised a lot of settings, what their default value should be.
+Many settings have had their defaults changed, mostly towards enabling modern feature by default.
```
```suggestion
Many settings have had their defaults changed, mostly towards enabling modern features by default.
```

michicc commented:
Someone™️ should add links to all the dev blogs to the appropriate sections.

TrueBrain commented:
Added all the links and made sure every blog post was covered (please check my work).

Also reworded some portions and added some text.

Collapsed "sandbox" with "new player experience". Not exactly correct, but having it in its own chapter also felt weird (it isn't that big of a new feature to warrant that, in my opinion).

Let me know what you think!

TrueBrain commented:
![News - 14 0 release](https://github.com/OpenTTD/website/assets/1663690/e65370d8-40b6-4821-83b0-aecc463a383a)

Steam:
```
Welcome to 14.0!

OpenTTD's first release was in March of 2004.
Now twenty years later, we are proud to present to you: release 14.0.

And boy, what a release it is.
Do I dare to say: this has been the biggest release yet?

Sometimes people try to tell us OpenTTD is dead.
Well, 14.0 shows you it really isn't.
There are so many new goodies in 14.0, that I don't even know where to start!
Let's go over a few, just to get you a feel for what to expect when you launch OpenTTD 14.0.
```

Discord / Reddit / Mastodon / ...
```
Welcome to 14.0!

OpenTTD's first release was in March of 2004.
Now twenty years later, we are proud to present to you: release 14.0.

And boy, what a release it is.
Do I dare to say: this has been the biggest release yet?

https://www.openttd.org/news/2024/04/01/openttd-14-0
```

stormcone commented on _posts/2024-04-01-openttd-14-0.md:
```diff
@@ -0,0 +1,125 @@
+
+## Survey support
+
+To finally put a rest to endless debates about what setting is used and which is not, OpenTTD not has an opt-in survey tool built-in.
+Over time this will allow us, developer, to better understand how OpenTTD is played, so we can better anticipate what will be a hit or a miss.
```
```suggestion
To finally put a rest to endless debates about what setting is used and which is not, OpenTTD now has an opt-in survey tool built-in.
Over time this will allow us, developers, to better understand how OpenTTD is played, so we can better anticipate what will be a hit or a miss.
```

stormcone commented on _posts/2024-04-01-openttd-14-0.md:
```diff
@@ -0,0 +1,125 @@
+Zephyris create a real (TTF) font heavily inspired by the sprite font.
+This means that scaling looks a lot better, especially for zoom levels like 1.5x.
+
+We love it so much, it is the default when you launch the game! (but no worries, you can revert to the sprite font if you really want to).
```
```suggestion
We love it so much, it is the default when you launch the game! (But no worries, you can revert to the sprite font if you really want to.)
```

stormcone commented on _posts/2024-04-01-openttd-14-0.md:
```diff
@@ -0,0 +1,125 @@
+## When one day isn't a day
+
+It is now possible to make the calendar time walk slower, up to a point it is frozen.
+This doesn't influence vehicle movement at all, nor the amount of goods transported, town growth etc.
```
```suggestion
This doesn't influence vehicle movement at all, nor the amount of goods transported, town growth, etc.
```

stormcone commented:
The last push has reverted the previous fixes.

Kuhnovic commented:
![Openttd_14_release](https://github.com/OpenTTD/website/assets/68320206/acf1e8ec-b9a5-4fd3-8e7b-6f2cd79e5a09)
[Openttd_14_release.zip](https://github.com/OpenTTD/website/files/14942565/Openttd_14_release.zip)

TrueBrain commented on _posts/2024-04-13-openttd-14-0.md:
```diff
@@ -0,0 +1,125 @@
+
+Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/02/17/openttd-truetype-fonts).
+
+## New players experience
```
This is the first one without a blog post, so it came in last.

Blame who-ever was too lazy to write a blog post about it :)

PeterN reviewed (approved):
tl;dr
